packages/checkout/core/src/components/plug-payments-session/plug-payments-session.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Sessions } from '../../services/sessions'
+import { PlugPaymentsError } from '../../types/plug-payments-error.types'
+import { PlugPaymentsSuccess } from '../../types/plug-payments-success.types'
+import { PlugPayments } from '../../types/plug-payments.types'
+
+export class PlugPaymentsSessionService implements PlugPayments {
+  readonly session?: Sessions
+
+  constructor() {
+    this.session = new Sessions()
+  }
```

Comment:
Quando for finalizar o restante da integração tem que lembrar de criar os atributos de `data`, `onPaymentSuccess` e `onPaymentFailed`. Igual nas outras services